tests/compile.sh

```diff
@@ -53,7 +53,7 @@ MAKE_THREADS=${MAKE_THREADS:-8}
 hostname
 
 set +x
-source $PATHTR/NEMS/src/conf/module-setup.sh.inc
+#source $PATHTR/NEMS/src/conf/module-setup.sh.inc
```

Comment:
Is this not needed anymore? If it isn't remove this line.

Reply:
It works for me on hera with our without the line. I can remove it, but we need to see what it does on other platforms. I think what is missing are the machine-dependent `module purge` commands (on some platforms like gaea you can't use `module purge`, on others you can and should). I'll remove it, we can put it back if needed.

Reply:
> It works for me on hera with our without the line. I can remove it, but we need to see what it does on other platforms. I think what is missing are the machine-dependent `module purge` commands (on some platforms like gaea you can't use `module purge`, on others you can and should). I'll remove it, we can put it back if needed.

One could also argue that these machine-dependent `module purge` commands should go into the module files where required.